### DISCUSSION TITLE
Why volumes do not have IDs?

### DISCUSSION THREAD

**Discussion by project0** [containers/podman#13903] [General]
I am currently working with the bindings and stumbled across some inconsistency of resource definitions. 

I wonder why volumes do not have any ID? 
Is name attribute enough to distinguish resources? But why do have all other resources IDs along to the name?

**Comment by mheon** [comment]
This is mainly a Docker compat thing - Docker volumes just use a name as an
identifier, without an ID, so we haven’t felt the need to add one to our
volumes. Why the original decision to have them be the only major type
without an ID was made, I have no idea.
